CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+# Essential Scala Collaborators
+
+Congratulations! You have been selected as a content owner for the  **Scala Essentials** nanodegree.
+
+
+## About the nanodegree
+
+The **Scala Essentials** nanodegree aims to provide a comprehensive tour of the Scala Programming Language for begginers. 
+
+You can see the design document [here](https://docs.google.com/document/d/1HZ1Rsn46x_BVAo8scSyVr9r5IesyTyHhGptjW1n_T6A/edit?usp=sharing).
+
+**Modules**:
+1. Introduction
+2. Essentials
+3. Objects and classes
+4. Functions
+5. Traits for data modelling
+6. Sequencing computation
+7. Idiomatic Scala
+8. Type classes
+9. Functional programming
+10. Popular frameworks and toolkits
+
+## About content owners
+
+As a content owner your job is to design and create the content of at least one module. You can build a small team with other content owners to assist on the content creation. Please consider that each conent owner has full responsibility of their own modules. 
+
+Content owners:
+* Mauricio Martín Saavedra Contreras
+* Rafael Avila 
+* Rodrigo Hernández Mota
+* Oscar Vargas Torres
+
+## About the content
+
+A module should have content for two different situtations:
+* **Documentation:** is a markdown file that contains the source knowledge, material, and resources needed to develop and present the module.
+* **Presentation:** is a markdown file that contains the presentation material used for the module slides. The slides are automatically created using [Pandoc](https://pandoc.org/) for the [Reavel.js presentation framework](https://revealjs.com/#/).
+
+The source content for each module in the markdown language is going to be safetly stored in a github repository. [Github pages](https://pages.github.com/) is used to publish the documentation and slides. 
+
+The particular content for each module can be found in the agenda of [the design document](https://docs.google.com/document/d/1HZ1Rsn46x_BVAo8scSyVr9r5IesyTyHhGptjW1n_T6A/edit?usp=sharing). 
+
+## Requirements
+
+**Note:** Collaborators running on Windows 10 are encoraged to try the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) (WSL). 
+
+As a content owner, you'll need the following tools in order to successfuly contribute to the project: 
+
+* The [git](https://git-scm.com/) open source distributed version control system.
+* A [github](http://github.com) account and access to the [Intersys Consulting](https://github.com/intersysconsulting) organization on the platform.
+* At least Java 8 and the Scala Build Tool [SBT](https://www.scala-sbt.org/1.x/docs/Setup.html).
+* The [Jekyll](https://jekyllrb.com/) site generator. 
+* [Pandoc](https://pandoc.org/), the file converter tool. 
+
+A setup script on a **debian-based OS** should look similar to:

```

Comment:
Is this script in an actual script file? Maybe having the script file can help a bit more in the setup process.

Reply:
Okay, I'll create a `setup/debian-based.sh` script with this contents. Maybe a Windows and Mac user can help us with a script for their platforms!